src/com/google/javascript/jscomp/CompilerOptions.java

```diff
@@ -2551,4 +2591,20 @@ public void addAlias(String alias, String definition) {
       }
     }
   }
+
+  /**
+   * An environment specifies the built-in externs that are loaded for a given
+   * compilation.
+   */
+  public static enum Environment {
+    /**
+     * Hand crafted externs that have traditionally been the default externs.
+     */
+    LEGACY,
```

Comment:
Yes. At this point we only have one set of externs. And they definitely depend on es6 types.